build_tools/travis/flake8_diff.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
```

Comment:
the file should start with the explanation of what's happening as you gave in this PR, I think.